internal/postrs/log.go

```diff
@@ -0,0 +1,33 @@
+package postrs
+
+import "C"
+
+import (
+	"github.com/spacemeshos/post/shared"
+)
+
+var log shared.Logger
+
+//export logCallback
+type logCallback func(level C.int, message *C.char)
```

Comment:
The other side is not included in post-rs yet and the API is not defined yet. Perhaps restrain from including it already until post-rs is ready?